minijinja/src/value/argtypes.rs

```diff
@@ -111,6 +111,9 @@ where
 /// Byte slices will borrow out of values carrying bytes or strings.  In the latter
 /// case the utf-8 bytes are returned.
 ///
+/// Similarly it's not possible to borrow dynamic slices
+/// ([`SeqObject`](crate::value::SeqObject)) into `&[Value]`.
+///
```

Comment:
Since `as_slice()` is gone, what is this referring to?

Reply:
The underlying `ArgTypes` cannot be implemented still. See the remaining internal uses of `as_slice`. I'm unsure how this can be resolved due to limitations with how lifetimes are entangled with value. Basically you cannot have a filter taking `foo(items: &[Value])` or `foo(items: Vec<i32>)` when the `items` value was a dynamic sequence.